docs/config.json

```diff
@@ -205,7 +205,7 @@
     },
     {
       "icon": "lock",
-      "title": "Configure Access",
+      "title": "Manage Access",
```

Comment:
Nit: Are we at all worried about the two similar titles seeming redundant? If not, ignore me.
![image](https://user-images.githubusercontent.com/2349184/218772517-3b0d73af-ea14-47e4-b962-f1e217f1a7be.png)

Reply:
I really, really want to reorganize the "Manage your Cluster" section, so hopefully we won't have this redundancy for long!